airgun/views/host_new.py

```diff
@@ -758,6 +759,36 @@ class insights(PF5Tab):
         )
         pagination = PF5Pagination()
 
+    @View.nested
+    class vulnerabilities(PF5Tab):
+        ROOT = './/div'
+
+        search_bar = SearchInput(locator='.//input[contains(@aria-label, "search-field")]')
+        cve_menu_toggle = PF5Button(".//button[contains(@class, 'pf-v5-c-menu-toggle')]")
+        no_cves_found_message = Text('.//h5[contains(@class, "pf-v5-c-empty-state__title-text")]')
+
+        vulnerabilities_table = pf5OUIAExpandableTable(
+            # component_id='OUIA-Generated-Table-2',
+            locator='.//table[contains(@class, "pf-v5-c-table")]',
+            column_widgets={
+                0: PF5Button(locator='.//button[@aria-label="Details"]'),
+                'CVE ID': Text('.//td[contains(@data-label, "CVE ID")]'),
+                'Publish date': Text('.//td[contains(@data-label, "Publish date")]'),
+                'Severity': Text('.//td[contains(@data-label, "Severity")]'),
+                'CVSS base score': Text('.//td[contains(@data-label, "CVSS base score")]'),
+            },
+        )
+        pagination = PF5Pagination()
+
+        @property
+        def is_displayed(self):
+            table_displayed = self.vulnerabilities_table.wait_displayed(exception=False)
```

Comment:
You can try either of these options.
```suggestion
            table_displayed = self.vulnerabilities_table.wait_displayed(timeout=60)
            or
            table_displayed = self.browser.wait_for_element(view.vulnerabilities_table, exception=False)
```